src/connector.ts

```diff
@@ -154,6 +154,7 @@ interface ConnectorOptions {
 // The Connector class is the main public API to interact
 // with the Cloud SQL Node.js Connector.
 export class Connector {
+  private closed = false;
```

Comment:
is closed really needed at the Connector level? Instance makes sense but what is the benefit for Connector level...

Reply:
This isn't necessary. I removed it.